modules/odm_georef/src/Georef.cpp

```diff
@@ -1185,9 +1185,39 @@ void Georef::chooseBestGCPTriplet(size_t &gcp0, size_t &gcp1, size_t &gcp2)
 
 void Georef::chooseBestCameraTriplet(size_t &cam0, size_t &cam1, size_t &cam2)
 {
+    size_t numThreads = 2;
```

Comment:
Very nice! In the future in might be useful to change numThreads from 2 to a function that retrieves the number of available CPUs of the machine.